src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -762,6 +763,116 @@ public function thisProductDoesNotRequireShipping(ProductInterface $product): vo
         $this->objectManager->flush();
     }
 
+    /**
+     * @Given product :product have the code equals :code
```

Comment:
See the comment in 16th line of `access_simple_product_show_page_from_products_index.feature` file